blog/2021-09-01-tdex-terraform-automation.md

```diff
@@ -56,6 +58,67 @@ $ ./deploy.sh
 ```
 ![Deploy](../static/img/deploy.png)
 
```

Comment:
Better to remeber at this point to take note of the IP address and user of the provisioned machine, will be used to configure the tdex cli